devel/external/AEGIS/README.md

```diff
@@ -0,0 +1,51 @@
+This README file describes the python code used to process AEGIS HST images in V and I bands to produce multi-band postage stamp images of galaxies. The selection procedure is explained in a separate document in devel/external/AEGIS/document/main.tex; a rendered version of the document may be viewed at https://www.overleaf.com/read/krbzsccfdbpm. 
+The final products are similar to the COSMOS Real Galaxy training sample.
+The script can be implemented to analyze any HST field data in multiple filters. It takes HST images in multiple bands, identifies objects with SExtractor, classifies those objects as stars and galaxies. Stars are then used to measure the PSF. Galaxies that satisfy certain criteria (see Section 3.8 in document) are selected for the main catalog. Separate catalogs are made for different bands. Each catalog can be individually used to initialize a galsim.RealGalaxyCatalog or galsim.COSMOSCatalog instance.  A postage stamp image is drawn for every galaxy in the catalog in each band. Each galaxy image will also have a postage stamp of its PSF in each band. The entries in different filter catalog files correspond to the same galaxies.
```

Comment:
I recommend saying a tiny bit more about PSF estimation.  For example "Stars are then used to measure the PSF by comparing with precomputed TinyTim PSFs and identifying the focus position that most closely matches this exposure.  Once the stars have been used to identify the focus position, the TinyTim PSF models are used for PSF estimation as a function of CCD position."  (I think that's what you did, right?)